source/Grid/Grid.js

```diff
@@ -472,6 +479,7 @@ export default class Grid extends Component {
         className={cn('Grid', className)}
         onKeyDown={this._onKeyPress}
         onScroll={this._onScroll}
+        onWheel={()=>{}}
```

Comment:
I don't understand why you need to specify anything for `onWheel`. Can you explain? Why is `onWheel={()=>{}}` any different/better than omitting the attribute entirely?